src/components/Navigation/Navigation.jsx

```diff
@@ -12,9 +12,9 @@ import Dropdown from '../Dropdown/Dropdown';
 import isClient from '../../utilities/is-client';
 
 // Load Styling
-import 'docsearch.js/dist/cdn/docsearch.css';
 import './Navigation.scss';
-import './Search.scss';
+import '@docsearch/css/dist/style.css';
+import 'prismjs/themes/prism-okaidia.css';
```

Comment:
@alexander-akait Do you have any idea why `import '@docsearch/css/dist/style.css'` won't import styles while `import 'prismjs/themes/prism-okaidia.css'` does? Seems like the `@` character was causing the problem.

Reply:
I know what's going on here, the `@docsearch/css` package has `sideEffects: false` in `package.json` causing the problem.